native_client/swift/deepspeech_ios/DeepSpeech.swift

```diff
@@ -30,7 +30,8 @@ public enum DeepSpeechError: Error {
     case failCreateSess(errorCode: Int32)
     case failCreateModel(errorCode: Int32)
 
-    // Additional case for invalid error codes, should never happen unless the user has mixed header and binary versions
+    // Additional case for invalid error codes, should never happen unless the
```

Comment:
Nice documenting. Do you know if we are ready to run swift-backed doc against readthedocs?

Reply:
Unfortunately there is no Swift support for our Sphinx version. So it'll be a bit more complicated.

Reply:
But I think it's supported by Doxygen, so we can use the existing support for that :)

Reply:
I looked for that too and couldn't find any Doxygen support. Do you have a link?

Reply:
Nope, I just saw a few people here and there that seemed to have it processing, as well as some apple github like https://github.com/apple/swift/blob/master/docs/doxygen.header

Reply:
That is for Doxygen docs of the Swift compiler, which is implemented in C++.